test/test_low_bit_optim.py

```diff
@@ -419,8 +419,8 @@ def test_optim_cpu_offload_save_load(self):
         for p1, p2 in zip(model1.parameters(), model2.parameters()):
             torch.testing.assert_close(p2, p1)
 
+    @parametrize("device", _DEVICES)
     def test_optim_bf16_stochastic_round_correctness(self):
```

Comment:
device arg is not added to the function

Reply:
@arlesniak  __DEVICES_ is the global variable in this file, I don't think your change is needed.

Reply:
The "test_optim_bf16_stochastic_round_correctness" should be parametrized with _DEVICES global variable, which contains other devices (XPU if detected). Without the PR changes the choice was cuda, cpu only.